### DISCUSSION TITLE
Summary

### DISCUSSION THREAD

**Discussion by drewskidang** [explosion/spaCy#13348] [Help: Coding & Implementations]
## How to reproduce the behaviour
 trying to extract the summary from the Llm task as annotation is there a config. Doc.text isn't doing anything 

## Your Environment

* Operating System:
* Python Version Used:
* spaCy Version Used:
* Environment Information:

**Comment by svlandeg** [comment]
Hi! Can you show us the config and code you're running, to better understand what you're trying to accomplish?
